corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -78,11 +78,21 @@
       {% trans "Learn more." %}
     </a>
   </p>
-  <div class="ko-loading">
-    <i class="fa fa-spinner fa-spin"></i>
-    {% blocktrans %}
-      Loading...
-    {% endblocktrans %}
+
+  <div id="datagrid-initial-loading"
```

Comment:
Should the class and ID of this node be named the same thing? I feel like that is slightly confusing.

Reply:
done